ert_gui/simulation/run_dialog.py

```diff
@@ -191,6 +194,36 @@ def on_new_iteration(self, parent: QModelIndex, start: int, end: int) -> None:
 
             self._tab_widget.addTab(widget, f"Iteration {iter}")
 
+    @Slot(QModelIndex)
+    def _job_clicked(self, index):
+        node = index.internalPointer()
+        if node is None or node.type != NodeType.JOB:
+            return
+
+        parent = node.parent
+        while parent.type != NodeType.ROOT:
+            if parent.type == NodeType.REAL:
+                real = parent.id
+            elif parent.type == NodeType.ITER:
+                iter = parent.id
+            parent = parent.parent
+
+        if index.column() == 4:
+            selected_file = node.data.get("stdout")
+        elif index.column() == 5:
+            selected_file = node.data.get("stderr")
```

Comment:
Same as above, 80% certain `index.data()` is the same thing.

So ```
selected_file = index.data()
```

given that you only consider columns that has files to be followed